connector/spanmetricsconnector/internal/metrics/metrics_test.go

```diff
@@ -103,3 +104,180 @@ func TestConnector_ExpoHistToExponentialDataPoint(t *testing.T) {
 		})
 	}
 }
+
+func TestSum_AddExemplar(t *testing.T) {
```

Comment:
I'd like to parallelise our tests, but I might do that for all spanmetrics connector tests in a separate PR, so nothing to change here for now.